packages/regenesis-surgery/scripts/contract-recompiler.ts

```diff
@@ -79,26 +84,27 @@ interface EtherscanContract {
       }
       try {
         const contractJson = JSON.parse(sourceCode)
-        console.log('got json', contractJson)
+        console.log('got json')
         if (contractJson.language) {
-          console.log('seems like multifile input', contractJson)
+          console.log('seems like multifile input')
           input = contractJson
         } else {
-          console.error('seems like just the source', contract.sourceCode)
+          console.error('seems like just the source')
           input.sources = contractJson
         }
       } catch (e) {
-        console.error('got error trying json', e)
+        console.error('got error trying json')
```

Comment:
these were a bunch of debug lines to help me figure out what was going on in the etherscan dump, so not attached to what they say and edit if you like!

Reply:
It was bloating the output a bunch and was hard to tell what was going on with the extra things logged, so i removed them. Eventually we probably want to remove all of them and be very strategic with the logging